### DISCUSSION TITLE
「好きな動画配信者は誰ですか？」 Please tell me your favorite streamer!

### DISCUSSION THREAD

**Discussion by masaki12-s** [NAIST-SE/AttractingContributors#411] [General]
あなたの好きな動画配信者を教えてください.
どんなジャンルでも結構です.
URLもどうぞ！

My fork: https://github.com/ruchan12/AttractingContributors-StreamerList

**Comment by aru1702** [comment]
日本語で説明していいですけど、英語でも後を説明するほうがいいですよ。
See my repo >> https://github.com/aru1702/AttractingContributors-TouhouFumo <<

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂
Here's my repo: https://github.com/Kotjik/AttractingContributors-FavoriteCharacter
